impress-org/givewp
# Refactor: update plans & pricing content in add-ons page

Resolves:
https://app.asana.com/0/1205251546311816/1205651762721231
https://app.asana.com/0/1205251546311816/1205651762721225


## Description

This PR updates the tab to which a user is navigated to from the Add-ons link.
Users are now directed to the Plans & Pricing tab. The order in which those plans display have also be rearranged.

Plan Order:
1. Plus 
2. Pro 
3. Basic 
4. Agency 

## Visuals


## Testing Instructions

- Click on the Addons link from the sidebar - verify you are directed to the plans & pricing tab.
- Verify new order of plans.

## Pre-review Checklist



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205849573153018
